siliconcompiler/tools/yosys/syn_asic.tcl

```diff
@@ -265,8 +268,14 @@ if {$script != ""} {
 foreach lib_file $sc_libraries {
     lappend abc_args "-liberty" $lib_file
 }
+set abc_dont_use []
+foreach lib "$sc_logiclibs $sc_macrolibs" {
+    foreach cell [dict get $sc_cfg library $lib asic cells dontuse] {
+        lappend abc_dont_use -dont_use $cell
+    }
+}
 
-yosys abc -liberty $sc_dff_library {*}$abc_args
+yosys abc {*}$abc_args {*}$abc_dont_use
```

Comment:
Not sure if removing the `-liberty` flag was intended.
In case you want to add it back in:
```suggestion
yosys abc -liberty $sc_dff_library {*}$abc_args {*}$abc_dont_use
```

Reply:
this is intentional, it's just passing in a library ABC doesnt actually need.